pycontw/pycontw-2021
# style: fix ios index layout

## Types of changes


* **Bugfix**

## Description

Fix images overlapping on index slogan and alter the slogan image spacing.

![](https://cdn.discordapp.com/attachments/753841328198254642/868049470774972436/image0.png)


## Steps to Test This Pull Request

1. Go to index page
2. Scroll to swiping photo section
3. Check layout issues

## Expected behavior

Layout with corresponding design

## Additional context

<img width="381" alt="截圖 2021-07-31 下午1 47 18" src="https://user-images.githubusercontent.com/13831865/127730191-52769556-a467-4b61-a652-9a63d6370215.png">
